src/components/Preview.vue

```diff
@@ -66,6 +75,7 @@ export default defineComponent({
         return {
             cffstr,
             copyToClipboard,
+            isMinimumCFF: computed(() => errors.value.map((v) => v.instancePath).some((i) => i.includes('title') || i.includes('authors'))),
```

Comment:
Also message. I think it will be too long for a one-liner, perhaps breaking lines in the chain will look better:
```javascript
errors.value
    .map((v) => v.instancePath)
    .some(...)
```